### ISSUE TITLE
WARNING: lock held when returning to user space!

### ISSUE THREAD

**Issue by marcin-github** [koverstreet/bcachefs#155]
```
[ 7744.223611] ================================================
[ 7744.223842] WARNING: lock held when returning to user space!
[ 7744.224074] 5.9.0-00313-g6d773fec1e10 #40 Tainted: G           OE   T
[ 7744.224322] ------------------------------------------------
[ 7744.224558] mass-check/29374 is leaving the kernel with locks still held!
[ 7744.224896] 1 lock held by mass-check/29374:
[ 7744.225127]  #0: ffffffff9306e320 (fs_reclaim){+.+.}-{0:0}, at: fs_reclaim_acquire.part.0+0x5/0x30
[19869.539454] Process accounting resumed
```

**Comment by koverstreet** [comment]
Found it! Fixed by 2f6fb2b0b7

**Comment by marcin-github** [comment]
Great! Thank you!
